packages/kit/src/runtime/client/renderer.js

```diff
@@ -160,6 +160,8 @@ export class Renderer {
 
 	/** @param {import('./types').NavigationTarget} selected */
 	notify(selected) {
+		dispatchEvent(new CustomEvent('sveltekit:navigationstart'));
```

Comment:
I wonder if we should go with something like `sveltekit:navigation-start`?

Reply:
good idea — changed